### ISSUE TITLE
[DropdownSkeleton] Does not have associated label or accessible name

### ISSUE THREAD

**Issue by zJag** [carbon-design-system/carbon#5613]
## Environment

> Browser

Found in Chrome. But probably all browsers.

> Automated testing tool and ruleset

DAP chrome extension

## Detailed description

> What version of the Carbon Design System are you using?

v10.10.1

> What did you expect to happen?

When using a DropdownSkeleton DAP should not find accessibility violations.

> What happened instead?

Two violations are found. 
- Form control element **div** is missing associated label.
- An interactive element/widget does not have an accessible name.

## Steps to reproduce the issue

1. Step one - Render DropdownSkeleton component
2. Step two - Inspect with DAP tool

[CodeSandbox](https://codesandbox.io/s/sweet-margulis-biggx)

## Additional information

This looks like it's because the skeleton is given a role of `button`([link](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Dropdown/Dropdown.Skeleton.js#L26)). If you look at the [TextInputSkeleton](https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/TextInput/TextInput.Skeleton.js), it does not have this "role" and does not have the same issues.
